var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -191,6 +194,9 @@ def configure_args(self):
         else:
             args.append("--with-kdreg=no")
 
+        if self.spec.satisfies("+uring"):
+            args.append("--with-uring=yes")
```

Comment:
For future reference, there are several helper methods for `AutotoolsPackage`s described at https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#helper-functions.